pkg/bytecode/vm_test.go

```diff
@@ -165,7 +165,8 @@ func TestVMArithmetic(t *testing.T) {
 					mustMake(t, OpSetGlobal, 0),
 				),
 			},
-		}, {
+		},
```

Comment:
I'm not sure how formatting managed to be off from the last change, but I ran `make fmt` in this PR  to resolve.

Reply:
I think I did that manually - i kinda dislike it when there are so many curly only lines spreading the data out so much and i've seen others do this too, but I'm not sure about this myself. I wonder though why `make fmt` didn't pick this up previously.